docs/fidelity-bonds.md

```diff
@@ -111,10 +154,112 @@ JoinMarket's coin control feature, so before spending you need to unfreeze the
+always disadvised, anyway) of timelocked addresses to any other wallets, as they use custom scripts.
+You must spend them from JoinMarket itself.
+
+### How many coins to lock up and for how long?
+
+Fidelity bonds UTXOs are valuable as soon as they confirmed. The simplified formula for a fidelity
+bond value with locked coins is:
+
+    bond_value = (locked_coins * exp(interest_rate * locktime))^2
+
+A few important things to notice:
+* The bond value goes as the _square_ of sacrificed value. For example if your sacrificed value is
+5 BTC then the fidelity bond value is 25 (because 5 x 5 = 25). If instead you sacrificed 6 BTC the
+value is 36 (because 6 x 6 = 36). The point of this is to create an incentive for makers to lump
+all their coins into just one bot rather than spreading it over many bots. It makes a sybil attack
+much more expensive.
+* The longer you lock for the greater the value. The value increases as the `interest_rate`, which
+is configurable in the config file with the option `interest_rate`. By default it is 1.5% per
+annum and because of tyranny-of-the-default takers are unlikely to change it. This value is probably
+not too far from the "real" interest rate, and the system still works fine even if the real rate
+is something like 3% or 0.1%.
+* The above formula would suggest that if you lock 3 BTC for 10000 years you get a fidelity
+bond worth `1.7481837557171304e+131` (17 followed by 130 zeros). This does not happen because the
+sacrificed value is capped at the value of the burned coins. So in this example the fidelity bond
+value would be just 9 (equal to 3x3 or 3 squared). This feature is not included in the above
+simplified equation.
+* After the locktime expires and the coins are free to move, the fidelity bond will continue to be
+valuable, but its value will exponentially drop following the interest rate. So it would be good
+for you as a yield generator to create a transaction with the UTXO spending it to another
+time-locked address, but it's not a huge rush (specifically, there's likely no need to pay massive
+miner fees, you can probably wait until fees are low).
+
+The full details on valuing a time-locked fidelity bond are [found in the relevant section of the
+"Financial mathematics of fidelity bonds" document](https://gist.github.com/chris-belcher/87ebbcbb639686057a389acb9ab3e25b#time-locked-fidelity-bonds).
+
+At any time you can use the orderbook watcher script to see your own fidelity bond value.
+
+Consider also the [warning on the bitcoin wiki page on timelocks](https://en.bitcoin.it/wiki/Timelock#Far-future_locks).
+
+I would recommend locking as many coins as you are comfortable with for a period of between 6
+months and 2 years. Perhaps at the very start lock for only 1 month or 2 months(?) It's a
+marketplace and the rules are known to all, so ultimately you'll have to make your own decision.
+
+### Creating watch-only fidelity bond wallets
```

Comment:
This part about watch-only FB wallets is confusing. In order to use FB with cold storage one needs a FB-only wallet with private keys (which is then locked away) and a regular (or FB?) segwit hot wallet that allows to import FB public keys.

The text however seems to document how to create a watch-only FB-only wallet that cannot be used for coinjoins and has no private keys? Is joinmarket going to load two wallets in that case, one non-FB segwit hot wallet and the FB-only watch-only wallet?

Reply:
Yes. In order to use FBs with cold storage, you need a FB wallet with private keys kept offline, and then on the online machine you create a FB-watch-only wallet and another regular segwit hot wallet. Then yield-generator will be provided with two wallet files on startup, one file for the hot wallet funds used for coinjoining and the other file for the watch-only fidelity bonds.